pkg/document/json/counter_test.go

```diff
@@ -19,7 +19,7 @@ package json_test
 import (
 	"math"
 	"testing"
-	time2 "time"
+	defaultTime "time"
```

Comment:
I try to suggest `gotime` instead of `defaultTime`.
The reason is this.
1. The Uber Style Guide and go blog recommends using all lowercase letters.
  - [User style guide - package names](https://github.com/uber-go/guide/blob/master/style.md?utm_source=gaerae.com&utm_campaign=%EA%B0%9C%EB%B0%9C%EC%9E%90%EC%8A%A4%EB%9F%BD%EB%8B%A4&utm_medium=social&fbclid=IwAR0EAuJQBl4SjfNknzGcC7dew691WQKLH7a0zasJ1tPn-9oNy5nUC4-RpHo#package-names)
  - [Go blog](https://blog.golang.org/package-names)
2. I referenced the `Renames should follow the same rules` rule of `rakyll.org` recommended in the Uber Style Guide.
  - [rakyll.org](https://rakyll.org/style-packages/)

Reply:
👍 Thanks for the comments. I modified the names of other packages as well.